cmd/bundle/generate/app.go

```diff
@@ -125,42 +106,3 @@ func NewGenerateAppCommand() *cobra.Command {
 
 	return cmd
 }
-
-func getAppConfig(ctx context.Context, app *apps.App, w *databricks.WorkspaceClient) (map[string]any, error) {
-	sourceCodePath := app.DefaultSourceCodePath
-
-	f, err := filer.NewWorkspaceFilesClient(w, sourceCodePath)
-	if err != nil {
-		return nil, err
-	}
-
-	// The app config is stored in app.yml or app.yaml file in the source code path.
-	configFileNames := []string{"app.yml", "app.yaml"}
```

Comment:
I don't quite understand how this matches description and NEXT_CHANGELOG.

The PR description says:
> We're suggesting to use app.yml files in app root instead of config section for apps

but this PR seems to remove support for app.yaml.

The NEXT_CHANGELOG entry says:
> Do not use app config section in test templates and generated app configuration ([#2599](https://github.com/databricks/cli/pull/2599))

Does "app config section" refers to app.yaml? Currently sounds like it refers to to apps section in databricks.yml.

Reply:
These lines of code were removing app.yaml from the app source code in favor of using app config section in bundle, now we don't remove app.yaml and don;t generate config section anymore because we ask users to rely on app.yaml. 

`Do not use app config section` is a section in bundle configuration.